app/controllers/user_bets_controller.rb

```diff
@@ -25,6 +27,18 @@ def create
     redirect_to root_path
   end
 
+  def destroy
+    @bet = Bet.find(@user_bet.bet_id)
```

Comment:
dùng tính năng của association đi em
```suggestion
    @bet = @user_bet.bet
```